CandidateReranking.md

```diff
@@ -118,7 +129,7 @@ A profile represents a candidate and is composed of data which, in relation to t
 | years_of_experience | Double | **FALSE** | Total years of experience the candidate has.
 | normalized_education_level | String[] | **FALSE** | Normalized Educations associated with the candidate i.e., Master's Degree, Bachelor's Degree etc.
 | unnormalized_education_level | String[] | **FALSE** | Unnormalized Educations associated with the candidate. i.e., masters degree, Master of Science, Masters etc.
-| major | String | **FALSE** | The major or field of study the candidate's degree pertains to.
+| majors | String[] | **FALSE** | The list of majors or fields of study the candidate's degree pertains to.
```

Comment:
As this is Json, it's more correct to say `array of majors` rather than `list of majors`.

Reply:
Yeah, array makes more sense. thanks.